docs/overview/Lifecycle.md

```diff
@@ -0,0 +1,157 @@
+renders a Holocron module and how you can configure the behavior of
+One App (and it's life cycles) via configuration.
+
+## 📖 Table of Contents
+* [One App Server](#one-app-server)
+  * [Holocron Runtime](#holocron-runtime)
+  * [Advanced Settings](#advanced-settings)
+* [Rendering](#rendering)
+
+## One App Server
+
+When the One App `node.js` server first boots up, there are a series of steps
+undertaken. The first step One App takes is configuration, which can be done via
+command line arguments and environment variables.
+
+**Configuration**
+
+Whenever One App is ran `NODE_ENV` must be defined with either
+`development` or `production`. One App behaves differently depending
+on which node environment it is running under. In development mode,
+One App spins up a development CDN and API server that run on
+`localhost` which `@americanexpress/one-app-runner` also utilizes.
+
+>  * [`NODE_ENV`](../api/server/environment-variables#node_env) ⚠️
+
+One App has a uniform value used to indicate which environment the app is
+running in. `ONE_CONFIG_ENV` is used internally as well as usable in our
+modules (to render per environment).
+
+>  * [`ONE_CONFIG_ENV`](../api/server/environment-variables#one_config_env) ⚠️
+
+The One App micro-frontend architecture utilizes an entry point to render
+the entire app. This entry point is our root Holocron module and its name
+can be set with `ONE_CLIENT_ROOT_MODULE_NAME`. Without this value, One App
+would not know which module to load and render from the module map (discussed below).
+
+>  * [`ONE_CLIENT_ROOT_MODULE_NAME`](../api/server/environment-variables#one_client_root_module_name) ⚠️
+
+Similar to `ONE_CONFIG_ENV`, you can specify which language pack
+environment name you wish to use using `ONE_CLIENT_LOCALE_FILENAME`:
+
+>  * [`ONE_CLIENT_LOCALE_FILENAME`](../api/server/environment-variables#one_client_locale_filename)
+
+**Production Environment**
+
+There are mandatory environment variables that need to be supplied to run One App
+and there are others to configure aspects of the One App runtime such as reporting
+URLs for errors and CSP violations that occurred on the client. You will need to
+include the environment variables below to run One App in production mode
+(`NODE_ENV === production`).
+
+>  * [`HOLOCRON_MODULE_MAP_URL`](../api/server/environment-variables#holocron_module_map_url) ⚠️
+>  * [`ONE_CLIENT_CDN_URL`](../api/server/environment-variables#one_client_cdn_url) ⚠️
+>  * [`ONE_CLIENT_REPORTING_URL`](../api/server/environment-variables#one_client_reporting_url) ⚠️
+>  * [`ONE_CLIENT_CSP_REPORTING_URL`](../api/server/environment-variables#one_client_csp_reporting_url) ⚠️
+
+### Holocron Runtime
+
+Before One App can start rendering HTML documents from the server, the app will
+preload every Holocron module defined in the Holocron module map. This module map
+is comprised of every module to be used by One App, including the root module which
+serves as our entry point.
+
+```json
```

Comment:
maybe link to the module map schema rather than an incomplete example?

Reply:
I've updated (10a7806)